src/together/resources/finetune.py

```diff
@@ -103,6 +103,9 @@ def create(
 
         assert isinstance(response, TogetherResponse)
 
+        # TODO: Remove it after the 21st of August
+        log_warn("The default value of batch size has been changed from 32 to 16 since together version >= 1.2.6")
```

Comment:
This is fine to have it before the response. A Jupyter Notebook or a Python script should highlight it from the return value.